src/authentication/views/RegisterOrLogin.scss

```diff
@@ -0,0 +1,17 @@
+.c-button.c-smartschool-button.c-button--primary {
+	background: url("/images/smartschool-login.png");
+	background-size: cover;
+  width: 293px;
+	height: 36px;
+	border: none;
+
+	&:hover {
+		background: url("/images/smartschool-login.png");
```

Comment:
Je kan hier een sass-mixin laten importen want die gebruik je twee maal in deze file.

```
@mixin background-smartschool {
	background: url("/images/smartschool-login.png");
	background-size: cover;
}
```

`@include background-smartschool;`

Maakt het niet korter maar dan moet je in de toekomst mogelijks niet op 2 plaatsen changen.